### ISSUE TITLE
No output to file

### ISSUE THREAD

**Issue by EternalDeiwos** [trentm/node-bunyan#448]
```js
const bunyan = require('bunyan')
let log = bunyan.createLogger({
  name: 'foo',
  streams: [
    {
      path: './mylog.json'
    }
  ]
})
log.fatal('this does not show in the file')
// => mylog.json is empty at this point.
```

So there seems to be some kind of bug where log data is not being written to files.
I used the above code to confirm this in the simplest possible manner; taken straight from your readme.

Could someone possibly tell me if I am missing something, or if this is a bug.
I'm using node 6.7.0 and bunyan 1.8.1

**Comment by jensvdh** [comment]
Seeing the exact same issue here. Bunyan 1.8.1 and node 5.8.0. Were you able to find a resolution?

**Comment by EternalDeiwos** [comment]
Not yet. I imagine the simplest way around it to be setting up my own stream and handling the file io myself. I've been putting it off, hoping for an answer here.

**Comment by jpaulin** [comment]
Adding a few lines into the stub file, what would it say? Ie. 

// Right before log.fatal gets called:
console.log('typeof log is: ' + typeof(log));
if (log) {
  console.log(' and typeof log.fatal is: ' + typeof(log.fatal));
} 

=> These should be
object
function 

Does it create the log object as above? 

It's fascinating problem, because in here, Node 6.7.0 and 1.8.1 of bunyan, it did work.

**Comment by EternalDeiwos** [comment]
I'm not sure what fixed it but as of 1.8.3 this seems to be a non-issue.
